### ISSUE TITLE
Seems to conflict with flutter_simple_permissions

### ISSUE THREAD

**Issue by zachgrayio** [babariviere/flutter_sms#37]
When attempting to use the `sms` library alongside `simple_permissions`, the app crashes as soon as `sms` initializes.

To reproduce, simply install `simple_permissions` to the `sms` example app and request a permission.

**Comment by babariviere** [comment]
Hi, are you requesting SMS permission with simple_permissions?

**Comment by zachgrayio** [comment]
@babariviere - nope, I'm just trying to get READ/WRITE access to external storage.

**Comment by babariviere** [comment]
Do you have any log?

**Comment by zachgrayio** [comment]
Sure, I'll fire that branch back up now

**Comment by babariviere** [comment]
Yes I think as we have 2 different listener for permission, it cannot choose which one to use

**Comment by zachgrayio** [comment]
I wonder how much work it would be to make use of simple permissions to request permissions in `sms` to avoid a conflict
